modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/service/impl/FragmentEntryLinkLocalServiceImpl.java

```diff
@@ -562,6 +584,46 @@ public FragmentEntryLink updateFragmentEntryLink(
 		return fragmentEntryLinkPersistence.update(fragmentEntryLink);
 	}
 
+	@Override
+	public void updateFragmentEntryLinksByRendererKey(
+			String rendererKey, String configuration, String css, String html,
+			String js, int type)
+		throws Exception {
+
+		if (Validator.isNull(rendererKey)) {
+			throw new UnsupportedOperationException();
+		}
+
+		StringBundler sb = new StringBundler(4);
+
+		sb.append("update FragmentEntryLink SET configuration = ?, css = ?, ");
+		sb.append("html = ?, js = ?, type_ = ?, lastPropagationDate = ? WHERE ");
+		sb.append("rendererKey = ? AND (configuration != ? OR css != ? OR ");
+		sb.append("html != ? OR js != ? OR type_ != ?)");
+
+		try (Connection connection = DataAccess.getConnection();
+			PreparedStatement preparedStatement = connection.prepareStatement(
+				sb.toString())) {
+
+			preparedStatement.setString(1, configuration);
+			preparedStatement.setString(2, css);
+			preparedStatement.setString(3, html);
+			preparedStatement.setString(4, js);
+			preparedStatement.setInt(5, type);
+			preparedStatement.setDate(6, null);
+			preparedStatement.setString(7, rendererKey);
+			preparedStatement.setString(8, configuration);
+			preparedStatement.setString(9, css);
+			preparedStatement.setString(10, html);
+			preparedStatement.setString(11, js);
+			preparedStatement.setInt(12, type);
+
+			preparedStatement.execute();
+		}
+
+		CacheRegistryUtil.clear();
```

Comment:
@jkappler this is way too overkilled. You should only clear FragmentEntryLink's entity/finder cache.

Reply:
Thanks @shuyangzhou, changed to 

```
fragmentEntryLinkPersistence.clearCache();

EntityCacheUtil.removeCache(FragmentEntryLink.class.getName()); 
```
Hope it's ok.